app/Services/RestApiClient.php

```diff
@@ -97,7 +99,7 @@ public function put($path, $body = [])
      */
     public function getJson($response, $asArray = false)
     {
-        return json_decode($response->getBody(), $asArray)->data;
+        return json_decode($response->getBody(), $asArray);
```

Comment:
For the errors... if we get an error response `data` doesn't exist... instead you have to request `->error` or even `->errors` ;) Better to let where it's actually being used grab what it needs and let the `getJson()` just turn the response into Json but not decide what property might contain the info you want.